test/test.extends.js

```diff
@@ -1,156 +1,173 @@
+const sinon = require('sinon')
 const Twig = require('../twig').factory();
 
 const {twig} = Twig;
 
 describe('Twig.js Extensions ->', function () {
-    it('should be able to extend a meta-type tag', function () {
```

Comment:
the change in this file is simply:

- nesting the existing tests under a describe blog specific for extending tags
- add test for custom filter (doesn't look like they were tested before, except in async tests)